File: leetcode/reverseInteger.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
    public:
    int reverse(int x) {
        string ax = to_string(x);
        reverse(ax.begin(), ax.end());
        cout << ax;
        return 3;
    }
};

int main() {
    Solution a = Solution();
    a.reverse(3);
}

File: leetcode/ReverseInteger.java
public class ReverseInteger {
    public int reverse(int x) {
        int res = 0;
        try {
            String tx = Integer.toString(Math.abs(x));
            String tx1 = new StringBuilder(tx).reverse().toString();
            res = (x / Math.abs(x)) * Integer.parseInt(tx1);
        } catch (Exception ex) { 
            res = 0;
        }
        return res;
    }

    public static void main(String[] args) {
        ReverseInteger re = new ReverseInteger();
        System.out.println(re.reverse(120));
        System.out.println(re.reverse(-120));
        System.out.println(re.reverse(2119999999));
    }
}
